test/e2e-test/py-test/test_rtmrs.py

```diff
@@ -0,0 +1,41 @@
+import pysdk_utils
+import logging
+import base64
+from ccnp import Measurement
+from ccnp import MeasurementType
+
+class TestRtmr:
```

Comment:
The tests are about getting and replay RTMRs. I think tests should be added for event logs, e.g. calling CCNP can fetch event logs, including IMA event logs.